randovania/games/samus_returns/logic_database/Area 2 Dam Exterior.txt

```diff
@@ -178,23 +174,37 @@ Extra - asset_id: collision_camera_005
   * Layers: default
   > Elevator to Area 2 Dam Entryway
       Trivial
-  > Door to Arachnus Arena
-      Trivial
   > Door to Rock Icicle Corridor
       Trivial
   > Door to Exterior Alpha+ Arena
       Diagonal Bomb Jump (Advanced) and Lay Bomb
+  > Next to Arachnus Arena
+      Trivial
 
 > Start Point (Top); Heals? False; Spawn Point
   * Layers: default
   * Extra - start_point_actor_name: ST_SG_Alpha_001
   > Door to Exterior Alpha+ Arena
       Trivial
 
-> Start Point (Bottom); Heals? False; Spawn Point
+> Next to Arachnus Arena; Heals? False; Spawn Point
```

Comment:
noticed this while reviewing a6: any reason this isnt a start point?

Reply:
Probably an oversight

Reply:
Oh, wait I see what you mean. Honestly, I want to rename all the nodes named `Start Point` to be more descriptive, because it can be confusing for the rooms that have multple starting locations, since the name that shows up is `Start Point`